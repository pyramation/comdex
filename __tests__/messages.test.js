import { comdex } from '../src/proto';

const MsgDepositRequest = comdex.vault.v1beta1.MsgDepositRequest;

it('messages.scoped', async () => {
  expect(
    MsgDepositRequest.fromPartial({
      amount: '23',
      from: 'from',
      id: 'id'
    })
  ).toMatchSnapshot();
});
