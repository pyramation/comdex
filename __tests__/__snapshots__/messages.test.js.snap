// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`messages.scoped 1`] = `
Object {
  "amount": "23",
  "from": "from",
  "id": Long {
    "high": 0,
    "low": 0,
    "unsigned": false,
  },
}
`;
